config/routes.rb

```diff
@@ -13,4 +13,9 @@
   resources :users do
```

Comment:
when I was poking around here, I realized we haven't created a `UsersController` yet, which Rails was complaining about. I think for now we need to tell Rails to skip over any users routes that it might be expecting by specifying `only: []`. That will tell Rails not to except any of the routes for showing/editing/deleting/etc. users until we're ready to add them.

```suggestion
  resources :users, only: [] do
```

Reply:
OK...so I adopted your suggestions as best I could.

There is still a problem.  The delete button works as expected (but now it's an ugly button so I'll need to fix the styling).  However I can not get error messages to show on the devise related forms.  That still seems like a problem with turbo and devise...and I'm not sure how to address it.

I would love a coding night! Or just shoot me in the right direction. I started a simple demo app using hotwire, to try to wrap my head around it a bit...still playing with that.

Reply:
Awesome, I think this looks good! I merged in some dependency updates to the master branch, so it looks like you need to either merge those in here or rebase with master to get up to date with those changes, then just run bundle install && yarn, and we should be good to merge!

Once those merge conflicts have been addressed, I’d be in favor of merging this as is and then doing the styling and digging into those error messages in new PRs.

I’ll take a look at my calendar and try to figure out another coding night, will hit you up on Slack!

Reply:
i gotta get a job so i use git all the time and better understand what I am doing!!

In any event, I think I have merged master and pushed up the changes....i think....i hope...